calling/src/commonMain/kotlin/com/wire/kalium/calling/CallClosedReason.kt

```diff
@@ -0,0 +1,22 @@
+package com.wire.kalium.calling
+
+enum class CallClosedReason(val avsValue: Int) {
+    NORMAL(avsValue = 0),
+    ERROR(avsValue = 1),
+    TIMEOUT(avsValue = 2),
+    LOST_MEDIA(avsValue = 3),
+    CANCELLED(avsValue = 4),
+    ANSWERED_ELSEWHERE(avsValue = 5),
+    IOERROR(avsValue = 6),
+    STILL_ONGOING(avsValue = 7),
+    TIMEOUT_ECONN(avsValue = 8),
+    DATA_CHANNEL(avsValue = 9),
+    REJECTED(avsValue = 10),
+    UNKNOWN(avsValue = -1);
```

Comment:
For future reference, I think we should map these guys to something "ours", as most of these errors are useless to the end user and clients consuming Kalium. Something like this, maybe:

```kotlin
sealed interface ReasonForClosingCall {

    /**
     * Call was finished by one of the participants
     */
    object Normal: ReasonForClosingCall

    /**
     * //I'm not sure this is correct: Call was dropped after ringing 
     * for too long without being picked-up
     */
    object Timeout: ReasonForClosingCall

    /**
     * [OneOnOne] call was picked up by this user on another device
     */
    object AnsweredByAnotherClient: ReasonForClosingCall

    /**
     * Receiver rejected the call
     */
    object Rejected: ReasonForClosingCall

    /**
     * Caller cancelled the call
     */
    object Cancelled: ReasonForClosingCall

    /**
     * Something unexpected happened, out of the control of the user
     */
    class Error(val rootCause: Throwable): ReasonForClosingCall // Can be an AvsException
}

class AvsException(message: String): RunTimeException(message)

// AVS-specific errors can be:
ReasonForClosingCall.Error(KaliumAVSException("Got DATA_CHANNEL(avsValue=9) for closing call))
```